src/components/explore/ExploreNavigation.tsx

```diff
@@ -0,0 +1,56 @@
+"use client";
+import Link from "next/link";
+import { ExploreNavigationItems } from "@/utils/data";
+import { usePathname } from "next/navigation";
+import { twMerge } from "tailwind-merge";
+import { ArrowLinkUpRight } from "@bitcoin-dev-project/bdp-ui/icons";
+
+const  ExploreNavigation = () => {
+  return (
+    <section className="max-md:hidden md:flex sticky top-0 flex-col flex-none gap-4  2xl:gap-6  w-full max-w-[240px] 2xl:max-w-[320px]">
+      <div className="flex flex-col gap-4 2xl:gap-6 rounded-lg px-5 py-[23px] border border-[#CCCCCC]">
```

Comment:
We have this color definition in the tailwind config, this can be fixed in a future PR, also in line 20